src/main/java/org/terasology/core/world/generator/facetProviders/SimplexHumidityProvider.java

```diff
@@ -63,11 +72,13 @@ public void process(GeneratingRegion region) {
         Border3D border = region.getBorderForFacet(SurfaceHumidityFacet.class);
         SurfaceHumidityFacet facet = new SurfaceHumidityFacet(region.getRegion(), border);
 
-        float[] noise = humidityNoise.noise(facet.getWorldRegion());
-        for (int i = 0; i < noise.length; ++i) {
-            noise[i] = TeraMath.clamp((noise[i] * 2.11f + 1f) * 0.5f);
+        Rect2i processRegion = facet.getWorldRegion();
+        for (BaseVector2i position : processRegion.contents()) {
+            // clamp to reasonable values, just in case
+            float noiseAdjusted = TeraMath.clamp(humidityNoise.noise(position.x(), position.y()) + .6f, 0f,1f);
```

Comment:
What are reasonable values, anyway? Not sure what the numbers here mean, may be good to name them via constants? There's also a space missing after `0f,` for a quick trivial style fix :-)